### ISSUE TITLE
File output is not supported in modelerfour

### ISSUE THREAD

**Issue by dolauli** [Azure/autorest#4745]
**Before filling a bug**

- [X] have you checked the [faq](https://github.com/Azure/autorest/tree/main/docs/faq.md) for known issues.
- [X] have you checked existing issues

The issue is original reported in https://github.com/Azure/autorest.powershell/issues/1149. After some investigation, I found the root cause is that file output seems not supported by modelerfour.

The response schema is defined as below.
```
      responses:
        '2XX':
          description: Success
          content:
            application/octet-stream:
              schema:
                type: object
                properties:
                  value:
                    type: string
                    format: base64url
                additionalProperties:
                  type: object
```

command used for code generation with the intermediate result generated.

```
autorest --powershell --use:@autorest/powershell@3.x --input-file:"https://gist.githubusercontent.com/peombwa/86bda868b27540fba3d9c08bf8f150f3/raw/a728cfe288156b301cd843b8556beb8539a13c85/Reports.yml" --output-folder:V3 --interactive
```

- [Link ](https://gist.githubusercontent.com/peombwa/86bda868b27540fba3d9c08bf8f150f3/raw/a728cfe288156b301cd843b8556beb8539a13c85/Reports.yml)to the swagger.

**Expected behavior**
Response is generated correctly with schema

**Actually result**
No schema is generated as below.

```
        responses:
          - language:
              default:
                name: ''
                description: Success
            protocol:
              http:
                statusCodes:
                  - 2XX
```

**Additional information**

The issue could be reproduced in the latest version of modelerfour - 4.26.2.

**Comment by timotheeguerin** [comment]
I don't think that swagger make sense. It is saying that the content type is `octet-stream` but it is a structured object. What is the object serialized as?
